.github/workflows/INTEGRATION_TEST.yml

```
@@ -2,12 +2,17 @@
 name: Integration test
 
 on:
+  push:
+    branches:
+      - release/*
   workflow_dispatch:
     inputs:
-      connectors_version:
-        description: 'The version of the connectors to test'
-        required: true
-        default: 'SNAPSHOT'
+      connectors-version:
+        description: 'The version of the connectors to test. If not provided, the version is determined based on the Maven project version. On main branch, the default is SNAPSHOT'
```

Suggestion: could also apply some 'branding'

```suggestion
        description: 'The version of the Connectors to test. If not provided, the version is determined based on the Maven project version. On main branch, the default is SNAPSHOT'
```